packages/block-editor/src/components/block-list/block.js

```diff
@@ -251,8 +252,9 @@ function BlockListBlock( {
 		}
 	}, [ isFirstMultiSelected ] );
 
+	const isTyping = useSelect( ( select ) => select( 'core/block-editor' ).isTyping() );
```

Comment:
This is buggy until we merge #16669

Reply:
So is this blocked by #16669?

Reply:
This could be rewritten to use an object (if we want to merge before that PR)